Update to minimist v1.2.3+

🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

Please read https://angular.io/guide/security#report-issues on how to disclose security related issues.

🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

`npm audit` command is reporting a new vulnerability in an angular/cli dev dependency. This does not appear critical but it does affect my company's internal tooling that requires passing `npm audit`s.

https://www.npmjs.com/advisories/1179 - moderate vulnerability in minimist pre v1.2.3

> if you add the following below devDependencies in package.json:
> 
> `"resolutions": { "minimist": "^1.2.3" }`
> 
> then delete node_modules and run
> `npx npm-force-resolutions && npm install`
> it will resolve the issue for current builds

I'm getting a npm ERR! extraneous: minimist@1.2.3 after doing your solution steps and running `npm ls minimist`

--- a/package.json
+++ b/package.json
@@ -114,7 +114,7 @@
     "husky": "^4.0.10",
     "jasmine": "^3.3.1",
     "jasmine-spec-reporter": "^4.2.1",
-    "karma": "~4.4.1",
+    "karma": "~5.0.0",
     "karma-jasmine": "~3.1.0",
     "karma-jasmine-html-reporter": "^1.4.0",
     "license-checker": "^25.0.0",

--- a/packages/angular_devkit/build_angular/package.json
+++ b/packages/angular_devkit/build_angular/package.json
@@ -90,7 +90,7 @@
     "jasmine-core": "~3.5.0",
     "jasmine-spec-reporter": "~4.2.1",
     "jquery": "^3.3.1",
-    "karma": "~4.4.1",
+    "karma": "~5.0.0",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~2.1.0",
     "karma-jasmine": "~3.1.0",

--- a/packages/schematics/angular/workspace/files/package.json.template
+++ b/packages/schematics/angular/workspace/files/package.json.template
@@ -33,7 +33,7 @@
     "codelyzer": "^5.1.2",
     "jasmine-core": "~3.5.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~4.4.1",
+    "karma": "~5.0.0",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~2.1.0",
     "karma-jasmine": "~3.0.1",

